### ISSUE TITLE
How can I inspect a Map?

### ISSUE THREAD

**Issue by smartcat32** [nalexn/ViewInspector#59]
```
struct MapSwiftUIView: View {
    
    @ObservedObject var viewModel: MapViewModel
    
    var body: some View {
        Map(coordinateRegion: self.$viewModel.region)
    }
}
```

I'd like to verify the Map is showing the region I gave to the MapViewModel (the center lat/lng and span) in my unit test.
sut.inspect().Map()  // no member 'Map'.  

Are the new SwiftUI views of iOS 14 not supported yet?

**Comment by nalexn** [comment]
Hey @smartcat32 , the `Map` and `MapAnnotation` views have fallen out of my scope because they were defined outside of the SwiftUI framework. I'll add them in one of the coming versions.
I wonder how can I scan all the iOS SDK frameworks for such one-off `Views`...

**Comment by nalexn** [comment]
I figured out that the easiest way to find all SwiftUI - companion frameworks are by looking into the iOS SDK content in Xcode's folder.
Here are the extensions for Xcode 12.2 that can be imported and explored right in the Xcode:

```swift
import _AVKit_SwiftUI
import _AuthenticationServices_SwiftUI
import _HomeKit_SwiftUI
import _MapKit_SwiftUI
import _QuickLook_SwiftUI
import _SceneKit_SwiftUI
import _SpriteKit_SwiftUI
import _StoreKit_SwiftUI
```
I'll update the [reference](https://github.com/nalexn/ViewInspector/blob/master/readiness.md) to reflect the currently not implemented SwiftUI APIs

**Comment by Tyler-Keith-Thompson** [comment]
I've got a [branch](https://github.com/Tyler-Keith-Thompson/ViewInspector/tree/map-support) I'm working on to add support for Maps. Hoping to submit a PR in a couple of days

**Comment by nalexn** [comment]
This is now available in [v0.8.0](https://github.com/nalexn/ViewInspector/releases/tag/0.8.0)
